sdk/textanalytics/Azure.AI.TextAnalytics/src/EntityRecognitionPiiTasksItem.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.AI.TextAnalytics
+{
+    [CodeGenModel("TasksStateTasksEntityRecognitionPiiTasksItem")]
+    public partial class EntityRecognitionPiiTasksItem
```

Comment:
please add a docstring here for the class either in this PR so we don't forget or when you do the analyze work :)